library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/MaskAllCheckBoxMapper.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder.mapper
+
+import android.widget.CheckBox
+import com.datadog.android.sessionreplay.internal.recorder.ViewUtils
+import com.datadog.android.sessionreplay.internal.utils.StringUtils
+import com.datadog.android.sessionreplay.model.MobileSegment
+
+internal class MaskAllCheckBoxMapper(
+    textWireframeMapper: TextWireframeMapper,
+    stringUtils: StringUtils = StringUtils,
+    uniqueIdentifierGenerator: UniqueIdentifierResolver =
+        UniqueIdentifierResolver,
+    viewUtils: ViewUtils = ViewUtils()
+) : CheckBoxMapper(textWireframeMapper, stringUtils, uniqueIdentifierGenerator, viewUtils) {
+
+    override fun resolveShapeStyle(view: CheckBox, checkBoxColor: String):
+        MobileSegment.ShapeStyle? {
+        return null
```

Comment:
why do we return `null` here? It is quite not obvious why having `shapeStyle = null` is related to masking all in `CheckBox`. Maybe we should add the comment here?

Same question for the `MaskAllCheckedTextViewMapper `.

Reply:
yes...we return null in order to not fill that checkbox...not showing the selection. I will add a comment.